indico/modules/events/contributions/models/contributions.py

```diff
@@ -494,6 +494,14 @@ def verbose_title(self):
     def paper(self):
         return Paper(self) if self._paper_last_revision else None
 
+    @property
+    def allowed_types_for_editable(self):
+        if not self.event.has_feature('editing'):
+            return []
+
+        submitted_for = [editable.type.name for editable in self.editables]
```

Comment:
could use a `{...}` set here

Reply:
I was using `set` initially but I didn't like the fact that I had to explicitly convert it to a `list` to make it serializable.

Reply:
But you never return this...

Reply:
Ahh.. ok, I thought you meant I should use sets in the entire function :P That is what I did before (sorry, i got confused)